<launch>
    
    <node name="occupancy_grid" pkg="planning" type="occupancy_grid.py" output="screen"/>
    <node name="send_goals" pkg="planning" type="send_goals.py" output="screen"/>
    <node name="Astar_planner" pkg="planning" type="Astar_planner.py" output="screen"/>

</launch>




